### DISCUSSION TITLE
Exploring Synergies for Sybil-Resistance and Private Biometrics

### DISCUSSION THREAD

**Discussion by Faresalfaress** [akash-network/community#680] [Integrations]
Hello Akash team, 

My name is Fares from Humanode.io , i'll give a short introduction on Humanode below: 
Humanode is the first cryptobiometric network where one human = one node = one vote that brings Sybil-resistance to the crypto industry using private biometric technology.

Built as an EVM Compatible Layer-1 with a novel Proof-of-Uniqueness and Proof-of-Existence consensus mechanism, Humanode is the most decentralized blockchain to date. Private facial recognition protects against multiple accounts and bots taking advantage of retroactive airdrops, DAOs, DeFi, NFT mints and GameFi. After just a year from mainnet launch, Humanode boasts 500,000+ users and 500+ projects utilizing its cryptobiometric tech.

Would love to explore potential synergies with Akash where we could offer our tech to your project/ecosystem. We have many use cases around Sybil-resistance such as filtering airdrops/contests from farmers and people abusing rewarding sistems in general. It could be implemented in governance if needed and anything you could think of that would need 1 person/1 account or one person-one vote scenarios.

Happy to answer any question and provide additional details. 

Regards,
Fares
Business developer
Humanode.io

**Comment by 31trainman** [comment]
This sounds interesting and I'd be interested in a chat on the Akash discord.

**Reply by Faresalfaress** [reply]
sent a message on Akash Discord :) Happy to have a chat

**Reply by 31trainman** [reply]
Did you tag @31trainman ?

**Reply by Faresalfaress** [reply]
Hey man, yes I did back then, happy to resume the conversation

**Reply by 31trainman** [reply]
What is your discord name?

**Reply by Faresalfaress** [reply]
@binance.whale
